frontend/src/App.css

```diff
@@ -41,6 +41,11 @@
   padding-bottom: 0 !important; /* overwrite DSFR default */
 }
 
+.fr-text--lead,
+.fr-text--lg {
+  margin-bottom: 2rem; /* overwrite DSFR default */
```

Comment:
DSFR margin bottom are really high (3rem for lead and 2.5rem for lg) it doesn't look great on our website